ref/android-features.md

```diff
@@ -0,0 +1,21 @@
+Anbox Cloud implements support for various features Android has. The following provides an overview which features are currently avilable as part of Anbox Cloud. If a feature is not listed, it is currently not supported.
```

Comment:
```suggestion
Anbox Cloud implements support for various features that Android has. The following table provides an overview of which features are currently available as part of Anbox Cloud. If a feature is not listed, it is currently not supported.
```

Reply:
"If a feature is not listed, it is currently not supported." - that is inconsistent, because there are also some rows without a check mark in the "Supported" column. Can we remove those rows? Then the sentence makes sense.

Reply:
> "If a feature is not listed, it is currently not supported." - that is inconsistent, because there are also some rows without a check mark in the "Supported" column. Can we remove those rows? Then the sentence makes sense.

Do we need those rows?

Reply:
@ru-fu I've updated the wording, can you check if you're fine now?

I want to explicitly call some features out as not being available and not just show a table with all green.